### ISSUE TITLE
question on `use_sid` v.s. `use_xvector` in multi-speaker TTS training

### ISSUE THREAD

**Issue by jefflai108** [espnet/espnet#4140]
Hi, 

I want to confirm my understanding of the difference between `use_sid` and `use_xvector` in multi-speaker TTS training? My guess is `use_xvector=true && use_sid=false` is using a pre-trained x-vectors to extract speaker embeddings for TTS training according to this [paper](https://arxiv.org/pdf/1806.04558.pdf). While setting `use_xvector=true && use_sid=true` is to use both x-vector speaker embeddings AND one-hot speaker embedding during training. 

Is my understanding correct? I see the pre-trained models only have `use_xvector=true`. Does turning both on make the synthesized speech better? 

thanks in advance!

**Comment by jefflai108** [comment]
(apology in advance for so many questions) I am also curious why the multi-speaker TTS recipe on LibriTTS is trained on [train-clean-460](https://github.com/espnet/espnet/blob/bfb23b87362869325bcb9e25358c375d4c320492/egs2/libritts/tts1/run.sh#L21) only instead of both `train-clean-460` AND 'train-other-500` from LibriTTS?

**Comment by kan-bayashi** [comment]
> Is my understanding correct? I see the pre-trained models only have use_xvector=true. Does turning both on make the synthesized speech better?

Correct. In that case, both x-vector and one-hot speaker id will be used.
But I've never tried that setting.

> I am also curious why the multi-speaker TTS recipe on LibriTTS is trained on [train-clean-460](https://github.com/espnet/espnet/blob/bfb23b87362869325bcb9e25358c375d4c320492/egs2/libritts/tts1/run.sh#L21) only instead of both train-clean-460 AND 'train-other-500` from LibriTTS?

train-other-500 seems dirty so it may affect the quality.
So I use only clean subset.
